crates/autopilot/src/arguments.rs

```diff
@@ -11,6 +11,11 @@ pub struct Arguments {
     #[clap(flatten)]
     pub shared: shared::arguments::Arguments,
 
+    /// A tracing Ethereum node URL to connect to, allowing a separate node URL
+    /// to be used exclusively for tracing calls.
+    #[clap(long, env)]
+    pub tracing_node_url: Option<Url>,
+
```

Comment:
Should we maybe make features using the tracing API depend on this argument?
Then we could drop `--skip-trace-api` and could always assume that `tracing_web3` actually supports the API as opposed to now where we fallback to using the `--node-url` for it which might not support the API?

Reply:
This is a nice idea! I like it.

Its a bit more "sensitive" since it requires configuration changes, but over-all, I think its a great change!

It also makes running our services with default arguments easier (i.e. you don't need to remember to always specify `--skip-trace-api`).

Wunderbar!

Reply:
Very nice!
Thank you 💪